local-tests/tests/testSignTransactionWithSolanaEncryptedKey.ts

```diff
@@ -0,0 +1,131 @@
+import { log } from '@lit-protocol/misc';
+import { TinnyEnvironment } from 'local-tests/setup/tinny-environment';
+import { signTransactionWithSolanaEncryptedKeyLitActionIpfsCid } from '@lit-protocol/wrapped-keys';
+import {
+  AccessControlConditions,
+  ILitNodeClient,
+  LitAbility,
+} from '@lit-protocol/types';
+import { encryptString } from '@lit-protocol/lit-node-client-nodejs';
+import {
+  Connection,
+  Keypair,
+  LAMPORTS_PER_SOL,
+  PublicKey,
+  SystemProgram,
+  Transaction,
+  clusterApiUrl,
+} from '@solana/web3.js';
+import {
+  LitAccessControlConditionResource,
+  LitActionResource,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from '@lit-protocol/auth-helpers';
+
+/**
+ * Test Commands:
+ * ✅ NETWORK=cayenne yarn test:local --filter=testSignTransactionWithSolanaEncryptedKey
+ * ✅ NETWORK=manzano yarn test:local --filter=testSignTransactionWithSolanaEncryptedKey
+ * ❔ NETWORK=localchain yarn test:local --filter=testSignTransactionWithSolanaEncryptedKey
+ */
+export const testSignTransactionWithSolanaEncryptedKey = async (
+  devEnv: TinnyEnvironment
+) => {
+  const alice = await devEnv.createRandomPerson();
+  const solanaKeypair = Keypair.generate();
+  const decryptionAccessControlConditions: AccessControlConditions = [
+    {
+      contractAddress: '',
+      standardContractType: '',
+      chain: 'ethereum',
+      method: '',
+      parameters: [':userAddress', 'latest'],
+      returnValueTest: {
+        comparator: '=',
+        value: await alice.wallet.getAddress(),
+      },
+    },
+  ];
+
+  const { ciphertext, dataToEncryptHash } = await encryptString(
+    {
+      accessControlConditions: decryptionAccessControlConditions,
+      dataToEncrypt: Buffer.from(solanaKeypair.secretKey).toString('hex'),
```

Comment:
Same comment as in the above file, why buffer?

Reply:
A Uint8Array is returned from Keypair.generate();, so I'm converting solanaKeypair.secretKey from a Buffer to a hex string